### PR TITLE
Test bytes_sent bugs.

### PR DESCRIPTION
**Pull request by mdw-at-linuxbox**
Rearrange logic to make it easier to measure accumulation.
Instrument the boto request/response loop to count bytes in and out.
Accumulate byte counts in usage like structure.
Compare actual usage reported by ceph against local usage measured.
Report and assert if there are any short-comings.

Nit: the logic to wait for "delete_obj" is not quite right.

Fixes: http://tracker.ceph.com/issues/19870
Signed-off-by: Marcus Watts <mwatts@redhat.com>

### CODE REVIEW

**Comment by mdw-at-linuxbox** [comment]
This tests the logic in https://github.com/ceph/ceph/pull/16834
I had originally thought this needed to go into a different repository, so that's why this is separate.

**Comment by smithfarm** [comment]
@mdw-at-linuxbox If you cherry-picked this commit into #16834, testing would be easier.

File: qa/tasks/radosgw_admin.py [hunk 1]
```diff
@@ -68,6 +267,12 @@ def task(ctx, config):
     """
     global log
 
+    # regions and config found from rgw task
```
Comment (oritwas) [inline qa/tasks/radosgw_admin.py]: Please use zonegroups not regions

File: qa/tasks/radosgw_admin.py [hunk 2]
```diff
@@ -78,10 +283,20 @@ def task(ctx, config):
     # choose first client as default
     client = clients_from_config[0]
 
+    multi_region_run = rgw_utils.multi_region_enabled(ctx)
```
Comment (oritwas) [inline qa/tasks/radosgw_admin.py]: multi_region_enable was removed from master

File: qa/tasks/radosgw_admin.py [hunk 3]
```diff
@@ -156,6 +380,376 @@ def task(ctx, config):
     assert out['keys'][0]['secret_key'] == secret_key
     assert not out['suspended']
 
+    # this whole block should only be run if regions have been configured
```
Comment (oritwas) [inline qa/tasks/radosgw_admin.py]: see above

**Comment by mdw-at-linuxbox** [comment]
I've made changes to radosgw_admin.py and I've merged that into PR # 16384.  So I'm going to close this PR.

**Pull request closed without merging**